resources/scripts/artifacts-api-latest-versions.sh

```diff
@@ -54,4 +54,8 @@ for version in ${QUERY_OUTPUT}; do
 done
 echo "}" >> "${TEMP_FILE}"
 
-jq . "${TEMP_FILE}" | tee ${OUTPUT}
+## As long as we need to support main, then let's
+## artificially duplicate the master document
+jq 'with_entries(select(.value.branch=="master")) | with_entries(.key = "main")' "${TEMP_FILE}" > "${TEMP_FILE}.main"
```

Comment:
Document format:

```json
{
  "6.8": {
    ..
  },
  "7.15": {
    ..
  },
  "7.16": {
    ..
  },
  "7.17": {
    ..
  },
  "8.0": {
    ..
  },
  "master": {
    ..
  }
}
```

so `main` is not in the list but required, for such, the jq with_entries trick appends the given document:


```
{
  "6.8": {
    ..
  },
  "7.15": {
    ..
  },
  "7.16": {
    ..
  },
  "7.17": {
    ..
  },
  "8.0": {
    ..
  },
  "master": {
    ..
  },
  "main": {
    ..
  }
}
```